common/koa-middleware/withToggles.js

```diff
@@ -33,16 +35,48 @@ const parseCookies = function(req) {
 function withToggles(ctx, next) {
   const cookies = parseCookies(ctx.req);
   const togglesCookies = cookies.filter(cookie =>
-    cookie.key.startsWith('toggle_')
+    cookie.key.startsWith(prefixName)
   );
   const toggles = togglesCookies.reduce((acc, cookie) => {
     return Object.assign({}, acc, {
-      [cookie.key.replace('toggle_', '')]: cookie.value === 'true',
+      [cookie.key.replace(prefixName, '')]: cookie.value === 'true',
     });
   }, {});
 
   ctx.toggles = { ...defaultToggleValues, ...toggles };
+  enableDisableToggler(ctx);
   return next();
 }
 
+function enableDisableToggler(ctx) {
+  const validToggle = (toggles, toggleItem) => {
+    return Object.keys(toggles).filter(toggle => {
+      return toggle === toggleItem;
+    });
+  };
+
+  if (ctx.toggles) {
+    // enable toggler of feature
+    if (ctx.query.toggler_test) {
```

Comment:
Could we stick to our `camelCase` style parameters?

Maybe prepare ourselves for making this a list, and use `toggles` which might be a CSV?

Reply:
We could also use the same syntax as the API, which is:
```
?toggles=buildingReopening,!modalFiltersPrototype
```

Reply:
ive done a first version where it can only set one at a time. but maybe after i could iterate and do this to include more..